solarforecastarbiter/datamodel.py

```diff
@@ -784,20 +816,6 @@ def __post_init__(self):
         __set_constant_values__(self)
         __check_axis_consistency__(self.axis, self.constant_values)
 
-    @classmethod
-    def from_dict(model, input_dict, raise_on_extra=False):
```

Comment:
changes to `_special_field_processing` make this unnecessary? I have no idea what's going on up there.

Reply:
changes to handling Union in _single_field_processing makes it unnecessary